tests/Fixer/ClassNotation/NoUnneededFinalMethodFixerTest.php

```diff
@@ -470,4 +470,32 @@ final public function test() {
 var_dump(Foo::Spades);',
         ];
     }
+
+    /**
+     * @dataProvider provideFix82Cases
+     *
+     * @requires PHP 8.2
+     */
+    public function testFix82(string $expected, string $input): void
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFix82Cases(): iterable
+    {
+        yield 'final-after-visibility' => [
```

Comment:
- Shouldn't it have `readonly` in the scenario's name?
- Don't we need scenario for `readonly final class` too? If fixer that reorders modifiers would be disabled, this one should still work.

Reply:
thanks, I've added more tests :+1: